README.md

```diff
@@ -0,0 +1,21 @@
+# Swift on Server Deployment Guide
+
+
+## General advise
+
+- Run production code in release mode by compiling with `swift build -c release`.
+- Until Swift 5.2 lands, always pass a `-Xswiftc -g` to get debugging symbols, otherwise your stacktraces will lack most symbol names.
+- If you have `--privileged`/`--security-opt seccomp=unconfined` containers or are running in VMs or even bare metal, you can run your binary with
+
+        lldb --batch -o run -k "image list" -k "register read" -k "bt all" -k "exit 134" ./my-program
+
+    instead of `./my-program` to get something something akin to a 'crash report' on crash.
+- If you don't have `--privileged` (or `--security-opt seccomp=unconfined`) containers (meaning you won't be able to use `lldb`) or you don't want to use lldb, consider using a library like [`swift-backtrace`](https://github.com/ianpartridge/swift-backtrace) to get stack traces on crash.
+- For best performance in Swift 5.2 (once released), pass `-Xswiftc -cross-module-optimization` (this won't work in Swift versions below 5.2)
```

Comment:
This will quickly become outdated, maybe it would be better to say "For best performance in Swift 5.2 or later"